bot/exts/pride/pride_facts.py

```diff
@@ -30,8 +28,7 @@ def __init__(self, bot: Bot):
     @staticmethod
     def load_facts() -> dict:
         """Loads a dictionary of years mapping to lists of facts."""
-        with open(Path("bot/resources/pride/facts.json"), "r", encoding="utf8") as f:
-            return json.load(f)
+        return json.loads(Path("bot/resources/pride/facts.json").read_text("utf8"))
```

Comment:
I'm going to keep commenting on these so you know where they are, but at this point you should know what should be done.